lib/compute/jenkins-main-node.ts

```diff
@@ -337,7 +337,7 @@ export class JenkinsMainNode {
           metrics_collected: {
             procstat: [
               {
-                pattern: 'jenkins',
+                pattern: 'Djenkins',
```

Comment:
NIT: can this be `-Djenkins`?

Reply:
not required, `Djenkins` should be fine.